composer.json

```diff
@@ -28,7 +28,7 @@
     "repositories": [
         {
             "type": "vcs",
-            "url": "https://github.com/madisoft/phpspec-data-provider-extension"
+            "url": "https://github.com/JellyBellyDev/phpspec-data-provider-extension"
```

Comment:
Why this change? What is the difference? php8 compatibility?

Reply:
Yep! I created the PR: https://github.com/madisoft/phpspec-data-provider-extension/pull/2/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R17  
In the meantime we test with my fork!  
Do you see any problems?

Reply:
I understand, ok for me 👍